src/main/webapp/app/course/courseTopicsList.xhtml

```diff
@@ -0,0 +1,40 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:h="http://java.sun.com/jsf/html"
+	xmlns:f="http://java.sun.com/jsf/core"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets">
+<h:head>
+	<title>Topic associati</title>
+</h:head>
+<h:body>
+	<h:form id="formTopicsOfCourse">
+		<h3>Topics of #{courseController.selectedCourse.name} course</h3>
+
+		<h:dataTable value="#{courseController.allTopics}" var="topic" border="0" id="courseTopicsTable">
+				<h:column>
+					<f:facet name="header">Id</f:facet>    				
+                	#{topic.id}
+            	</h:column>
+
+				<h:column>
+					<f:facet name="header">Name</f:facet>
+                	#{topic.name}
+            	</h:column>
+
+				<h:column>
+					<f:facet name="header">Description</f:facet>
+                	#{topic.description}
+            	</h:column>
+
+				<h:column>
+					<f:facet name="header">Selection</f:facet>
+					<h:selectBooleanCheckbox id="selectionTopic" value="#{topic.checked}"/>
+		 		</h:column>
+		</h:dataTable>
+		
+		<h:panelGrid id="panelButtonsUpdateTopic" border="0" columns="2">
+			<h:commandButton id="aggiornaTopic" value="Update Topic" action="#{courseController.updateCourseTopics()}" />
+			<h:commandButton id="tornaIndietro" value="Back" action="#{courseController.cleanTopic(selectedCourse)}" />
```

Comment:
In the cleanTopic you shouldn't pass anything. As the name indicates you should only clean allTopics from the "checked" ones.  In fact better to change the name in: cleanAllTopics()

Reply:
done